src/main/java/it/pagopa/transactions/commands/handlers/TransactionActivateHandler.java

```diff
@@ -76,7 +76,7 @@ public TransactionActivateHandler(
             ConfidentialMailUtils confidentialMailUtils,
             @Value("${azurestorage.queues.transientQueues.ttlSeconds}") int transientQueuesTTLSeconds,
             @Value("${nodo.parallelRequests}") int nodoParallelRequests,
-            TracingUtils tracingUtils,
+            it.pagopa.ecommerce.commons.queues.TracingUtils tracingUtils,
```

Comment:
Use import 🙂

Reply:
Woops, my bad, missed it 😌 Fixed with https://github.com/pagopa/pagopa-ecommerce-transactions-service/pull/289/commits/a09db13566d4786b5942b95671cd5014a346e380